docs/resources/ec_deployment.md

```diff
@@ -97,98 +99,98 @@ resource "ec_deployment" "ccs" {
 
 The following arguments are supported:
 
-* `region` - (Required) ESS region where to create the deployment. For ECE environments "ece-region" must be set.
+* `region` - (Required) Elasticsearch Service (ESS) region where to create the deployment. For Elastic Cloud Enterprise (ECE), set the `"ece-region"` environment.
 
--> Changing the `region` will cause the resource to be destroyed and re-created.
+-> If you change the `region`, the resource is destroyed and re-created.
```

Comment:
I'm not sure the tense is the most accurate since users have the ability to see what happens with `terraform plan` or `terraform apply`, I think `the resource will be destroyed and re-created` fits better with the behavior of terraform and what users see?